interfaces/admin_cli.py

```diff
@@ -0,0 +1,501 @@
+            if username == '#':
+                ManageVolunteerMenu(self.user).cmdloop()
+            try:
+                find_volunteer = volunteer_controller.find_volunteer(username)
+                volunteer_controller.reactivate_volunteer(find_volunteer)
+                print("\x1b[6;30;42m success! \x1b[0m")
+                print(f"Volunteer {username} reactivated.")
+                self.return_previous_page()
+                break
+            except ControllerError:
+                print(f"\033[31m* Volunteer {username} not found. Please check and re-enter.\033[00m")
+                continue
+
+    def do_delete_volunteer(self, arg):
+        """
+        #6 Delete a volunteer
+        """
+        while True:
+            username = input("Enter the volunteer's username to delete (Press # to leave this page):")
+            if username == '#':
+                ManageVolunteerMenu(self.user).cmdloop()
+            try:
+                find_volunteer = volunteer_controller.find_volunteer(username)
+                volunteer_controller.delete_volunteer(find_volunteer)
+                print("\x1b[6;30;42m success! \x1b[0m")
+                print(f"Volunteer {username} deleted.")
+                self.return_previous_page()
+                break
+            except ControllerError:
+                print(f"\033[31m* Volunteer {username} not found. Please check and re-enter.\033[00m")
+                continue
+
+    def do_return_main_menu(self, arg):
+        AdminShell(self.user).cmdloop()
+
+
+class ManageRefugeeMenu(AdminShell):
+    refugee_menu = {"logout": 0, "create_refugee": 1, "view_refugee": 2,
+                    "return_main_menu": "r"}
+
+    def plan_menu(self) -> None:
+        print("\n\033[100m\033[4m\033[1mManage Refugee Profile\033[0m\n"
+              "[ 1 ] Create a refugee profile\n"
+              "[ 2 ] View a refugee profile\n\n"
+              "[ R ] Return to Main Menu\n")
+
+    def preloop(self) -> None:
+        """
+        Display plan menu.
+        :return:
+        """
+        self.plan_menu()
+
+    def precmd(self, option: str) -> str:
+        """
+        Transfer option numbers to function name for Admin Menu
+        """
+        if option.isdigit() and int(option) in list(self.refugee_menu.values()):
+            return list(self.refugee_menu.keys())[list(self.refugee_menu.values()).index(int(option))]
+        elif option.upper() == 'X':
+            return list(self.refugee_menu.keys())[list(self.refugee_menu.values()).index(option.lower())]
+        else:
+            return "invalid_input"
+
+    # ----- basic commands -----
+    def do_create_refugee(self, arg):
+        """
+        #1 Create a refugee profile
+        """
+        print("\n\033[100m\033[4m\033[1m{}\033[0m ".format("Create a new refugee profile"))
+        r_firstname = input("Enter refugee's first name:")
+        r_lastname = input("Enter refugee's last name:")
+        r_camp = input("Enter refugee's camp:")
```

Comment:
Do we need separate logic for admin and volunteer here? 
Cuz for volunteers, I believe that they can only register refugees to the camp they are currently assigned to.

Reply:
I think we should, and this should be done in `refugee controller` in the same way as edit_camp() in #34

Reply:
Fixed in new version